README.md

```diff
@@ -1,19 +1,100 @@
 # Submit a GAR Public Site UI
 
-## Docker Usage
-The following depends on a .env file in the root of this project with the required environment variables. This will start a local test instance of keycloak for local testing.
+## Requirements
+- docker
+- docker-compose
+- node
+- npm
+- your email added to GOV Notify Dev whitelist by repository administrator
+
+## Development
+
+The following depends on a `.env.dev` file in the root of this project with
+the required environment variables. To obtain it please contact repository administrator.
+
+This will start local instances of:
+- Database (used for both Frontend and Backend)
+- API (you need a local copy of `api` docker image loaded, 
+       please contact repository administrators to obtain it)
+- ClamAV engine
+- ClamAV REST API 
+- Frontend
+
+Please note - due to security configuration, the following lines need to be 
+commented out for local development, but not committed back to the repository:
+```
+./src/server.js (lines 95-98):
```

Comment:
When I test the UI, the radio buttons don't work. Eg. when i choose "online" and press continue, it takes forever and then i get error "This page isn't working." the links have broken.

Reply:
This should be working now. The issue was that the code actually checks the **text** of the radio button option selected. So when I changed the text in the seed data this broke the page!